dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/options/src/com/ibm/ws/jaxrs/fat/options/OptionsTestServlet.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs.fat.options;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
Some of the tests might read easier if we use `assertFalse(condition)` instead of `assertTrue(!condition)`.
```suggestion
import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;
```